### ISSUE TITLE
-version should require -eula

### ISSUE THREAD

**Issue by johnseekins** [VictoriaMetrics/VictoriaMetrics#1621]
**Describe the bug**
When running the enterprise version of cluster tools, asking for the application version shouldn't require accepting the eula.

**To Reproduce**
```
# vmselect-prod -version
Copyright 2018-2021 VictoriaMetrics, Inc - All Rights Reserved.
For any questions please contact info@victoriametrics.com
Pass -eula command-line flag if you have an enterprise license for running this software
See https://victoriametrics.com/assets/VM_EULA.pdf for details

# vmselect-prod -version -eula
Copyright 2018-2021 VictoriaMetrics, Inc - All Rights Reserved.
For any questions please contact info@victoriametrics.com
vmselect-20210901-143424-tags-v1.65.0-enterprise-cluster-0-gf2d969c73
```

**Expected behavior**
`<application> -version` would return the version regardless of whether the tool is OSS or Enterprise.

This particularly makes deploying Enterprise on top of OSS installs through tools like Ansible difficult because I have to somehow run the version command in two different ways during the deployment.

**Comment by valyala** [comment]
The issue has been fixed in the development branch for cluster version. The fix will be included in the next release of VictoriaMetrics.

**Comment by johnseekins** [comment]
Excellent. Thanks for fixing that, @valyala:
```
# vmselect-prod -version
vmselect-20210920-124601-tags-v1.66.0-enterprise-cluster-0-g70c3c50cb

**Comment by valyala** [comment]
FYI, the bugfix has been included in VictoriaMetrics enterprise v1.66.0.
